zcash_proofs/src/sapling/verifier.rs

```diff
@@ -1,37 +1,37 @@
-use bellman::{
-    gadgets::multipack,
-    groth16::{verify_proof, PreparedVerifyingKey, Proof},
-};
+use bellman::{gadgets::multipack, groth16::Proof};
 use bls12_381::Bls12;
 use group::{Curve, GroupEncoding};
 use zcash_primitives::{
-    constants::{SPENDING_KEY_GENERATOR, VALUE_COMMITMENT_RANDOMNESS_GENERATOR},
     sapling::redjubjub::{PublicKey, Signature},
     transaction::components::Amount,
 };
 
 use super::compute_value_balance;
 
+mod single;
+pub use single::SaplingVerificationContext;
+
+mod batch;
+pub use batch::BatchValidator;
+
 /// A context object for verifying the Sapling components of a Zcash transaction.
-pub struct SaplingVerificationContext {
+struct SaplingVerificationContextInner {
     // (sum of the Spend value commitments) - (sum of the Output value commitments)
     cv_sum: jubjub::ExtendedPoint,
-    zip216_enabled: bool,
 }
 
-impl SaplingVerificationContext {
+impl SaplingVerificationContextInner {
     /// Construct a new context to be used with a single transaction.
-    pub fn new(zip216_enabled: bool) -> Self {
-        SaplingVerificationContext {
+    fn new() -> Self {
+        SaplingVerificationContextInner {
             cv_sum: jubjub::ExtendedPoint::identity(),
-            zip216_enabled,
         }
     }
 
     /// Perform consensus checks on a Sapling SpendDescription, while
     /// accumulating its value commitment inside the context for later use.
     #[allow(clippy::too_many_arguments)]
-    pub fn check_spend(
+    fn check_spend<C>(
```

Comment:
Do these visibility changes need to be noted in the changelog?

Reply:
No. Note that I changed the name of the struct these functions are in, and the old struct name (which is the "new" struct in `single.rs`) has the old function signatures with the same public visibility.